Sources/Fuzzilli/Fuzzer.swift

```diff
@@ -282,8 +301,16 @@ public class Fuzzer {
             didCrash = true
 
         case .succeeded:
-            if let aspects = evaluator.evaluate(execution) {
+
+            if let aspects = evaluator.evaluate(execution){
```

Comment:
With this logic, when `includeAll` is selected, doesn't it now still minimize programs that find new coverage? That's probably not what we want though, is it?

Reply:
Would there be value in adding both the minimized and full versions to the corpus? The minimized version that expands coverage would be valuable as input to the combine & concat mutators, while the full, unminimized one is a good starting point for variants. What do you think?

Reply:
Thinking about this a bit more, I think keeping only the full version makes sense here, to keep the behavior clear. My current setup is having an engine-unique corpus made of regressions for that engine, and a shared corpus of all regression tests, which are used as input to importCorpusAll and importCorpusNewCov, respectively. Including a sample program in both corpora ensures the behavior I mentioned in my previous comment.

Reply:
Yeah, that sounds good!